go.mod

```diff
@@ -5,7 +5,8 @@ go 1.16
 require (
 	github.com/client9/misspell v0.3.4
 	github.com/golangci/golangci-lint v1.25.1
-	github.com/google/go-github/v39 v39.0.0
+	github.com/google/go-github/v39 v39.2.0
```

Comment:
Should this be removed in favour of the version below?

Reply:
@jcudit I'm so sorry, I left out the second step! :facepalm: It's all updated now and is ready for re-review. Please forgive me for the hassle.